verification/tagTCI.go

```diff
@@ -8,16 +8,14 @@ import (
 	"testing"
 )
 
-// This file is used to test the tagTCI command.
-
 func TestTagTCI(d TestDPEInstance, client DPEClient, t *testing.T) {
-	// Try to create the default context if isn't done automatically.
-	if !d.GetSupport().AutoInit {
-		handle, err := client.InitializeContext(InitIsDefault)
-		if err != nil {
-			t.Fatalf("Failed to initialize default context: %v", err)
-		}
-		defer client.DestroyContext(handle, DestroyDescendants)
+	testTagTCI(d, client, t, false)
+}
+
+func testTagTCI(d TestDPEInstance, client DPEClient, t *testing.T, simulation bool) {
+	ctx := getContextHandle(d, client, t, simulation)
+	if simulation {
+		defer client.DestroyContext(ctx, DestroyDescendants)
```

Comment:
same problem here, handle is initialiaed but never used, also its scope is only within the IF block